jmclient/jmclient/yieldgenerator.py

```diff
@@ -159,6 +161,28 @@ def on_tx_confirmed(self, offer, confirmations, txid):
                 confirm_time / 60.0, 2), ''])
         return self.on_tx_unconfirmed(offer, txid, None)
 
+    def get_available_mixdepths(self, verbose=True):
```

Comment:
I think it should be `verbose=False` by default. `verbose` is not implemented right now but it would probably add some logging output that would usually not be needed, especially for a yg.

Reply:
I'm happy to change it, but I just used what the wallet function itself has: https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/3af6f43da280fed3077f715848cc3b6d2f1f32d1/jmclient/jmclient/wallet.py#L703

Reply:
Ah, yes, I see. I still think for YG it should be `False` by default.

Reply:
Ok.  I've set it to false now everywhere in the YG.